src/20_Components/amp-font.html

```diff
@@ -0,0 +1,114 @@
+<!-- 
+  #### Introduction
+
+  Learn how to use the `amp-font` component to trigger and monitor the loading of custom fonts on AMP pages.
+
+-->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="https://ampbyexample.com/components/amp-font/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-font" src="https://cdn.ampproject.org/v0/amp-font-0.1.js"></script>
+  <!-- 
+    Define custom CSS rules to style your document depending on whether a font was loaded successfully or not. Find out about all the supported css classes [here](https://github.com/ampproject/amphtml/blob/b1607dff9446d332afcdead64b63e8d25e74cff7/extensions/amp-font/amp-font.md)
+  -->
+  <style amp-custom>
+    @font-face {
+      font-family: 'Font Does Not exist';
+      font-style: normal;
+      font-weight: 400;
+      src: url(fonts/FontDoesNotExist.ttf) format('truetype');
+    }
+    @font-face {
+      font-family: 'Another Font That Does Not exist';
+      font-style: normal;
+      font-weight: 400;
+      src: url(fonts/FontDoesNotExist.ttf) format('truetype');
+    }
+    @font-face {
+      font-family: 'Tangerine';
+      font-style: normal;
+      font-weight: 400;
+      src: local('Tangerine'), url(http://fonts.gstatic.com/s/tangerine/v7/HGfsyCL5WASpHOFnouG-RJBw1xU1rKptJj_0jans920.woff2) format('woff2');
+    }
+
+    .font-does-not-exist-loaded .font-does-not-exist {
+      font-family: 'Font Does Not exist', serif, sans-serif;
+    }
+    .tangerine-loaded .tangerine {
+      font-family: 'Tangerine', serif, sans-serif;
+    }
+    .another-font-does-not-exist-loaded .another-font-does-not-exist {
+      font-family: 'Another Font That Does Not exist', serif, sans-serif;
+    }
+    .font-does-not-exist-loading, .tangerine-loading, .another-font-does-not-exist-loading {
+      color: #00f;
+    }
+    .font-does-not-exist-loading .font-does-not-exist, .tangerine-loading .tangerine, .another-font-does-not-exist-loading .another-font-does-not-exist {
```

Comment:
formatting nit: I think formatting it like this makes it easier to read:

```
font-does-not-exist-loading .font-does-not-exist, 
.tangerine-loading .tangerine,
 .another-font-does-not-exist-loading .another-font-does-not-exist {
     color: #0ff;
}
```

Same applies to the rule below.